code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -74,7 +74,8 @@
 	flags_1 = CONDUCT_1
 	sharpness = SHARP_EDGED
 	w_class = WEIGHT_CLASS_NORMAL
-	force = 17
+	force = 24 //SKYRAT EDIT - ORIGINAL: 17
+	armour_penetration = 15 // SKYRAT EDIT - ADDITION
 	throwforce = 10
```

Comment:
can i suggest upping this to 17 as well with the plus 7 buff?

Reply:
at the moment, i'd prefer their blades stay on the weaker side in raw 'traditional' damage, due to the extra effects they stack on later